app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+from app.errors import NotWearingMaskError, VaccineError
+from app.cafe import Cafe
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    masks_to_buy = 0
+    result = ""
+
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except NotWearingMaskError:
+            masks_to_buy += 1
+        except VaccineError as e:
+            result = str(e)
+
+    if masks_to_buy and not result:
+        result = f"Friends should buy {masks_to_buy} masks"
```

Comment:
I don't quite understand why you need a variable result
in the 17 line you can check if mask_to_buy > 0 : return f"string"
and final return only f"string" without OR